ipa-core/benches/oneshot/ipa.rs

```diff
@@ -100,11 +100,14 @@ async fn run(args: Args) -> Result<(), Error> {
     let _prep_time = Instant::now();
     let config = TestWorldConfig {
         gateway_config: GatewayConfig::new(args.active()),
+        extra_log_target: option_env!("CARGO_CRATE_NAME"),
```

Comment:
I wonder if it is worth just setting or changing the `RUST_LOG` env variable here, may keep the logging API cleaner and more maintainable if we ever need to add 2nd, 3rd target etc